src/App.tsx

```diff
@@ -1,10 +1,50 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useMemo, useState } from 'react';
 import { UserWarning } from './UserWarning';
+import { getTodos } from './api/todos';
+import { Todo } from './types/Todo';
+import { TodoList } from './components/TodoList/TodoList';
+import { TodoFooter } from './components/TodoFooter/TodoFooter';
+import { Status } from './types/Status';
 
-const USER_ID = 0;
+const USER_ID = 12042;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [error, setError] = useState<string | null>(null);
+  const [status, setStatus] = useState<Status>(Status.all);
+
+  useEffect(() => {
+    getTodos(USER_ID)
+      .then(setTodos)
+      .catch(() => (setError('Unable to load todos')))
+      .finally(() => (
+        setTimeout(() => {
+          setError(null);
+        }, 3000)
+      ));
```

Comment:
Why do you set a timer in the finally block, in your case you will do it even if there is no error.

You can create a separate function to handle errors
```javascript
const handleError = (error: string) => {
  // set error
  // set timer to remove error in 3 second
};
```